spec/support/medium_controller_spec.rb

```diff
@@ -0,0 +1,86 @@
+RSpec.shared_examples "a medium controller" do
+  describe "GET 'index'" do
+    it "is successful" do
+      get :index
+      expect(response.status).to eq 200
+    end
+  end
+
+  describe "GET 'show'" do
+    it "renders the show view" do
+      medium = model.create(name: "Test")
```

Comment:
It might be a good idea to use `good_params[model.name.to_sym]` here instead of passing `name` directly.

This way, if the design changes such that more than just the `name` field is required, you'll only need to update the `good_params` hash to ensure that these dummy model instances are being created correctly.